awscrt/auth.py

```diff
@@ -306,6 +306,63 @@ def new_delegate(cls, get_credentials):
         binding = _awscrt.credentials_provider_new_delegate(get_credentials)
         return cls(binding)
 
+    @classmethod
+    def new_cognito(
+            cls,
+            endpoint,
```

Comment:
trivial/debatable: I'd suggest using keyword-only arguments like
```
def new_cognito(
    cls,
    *,
    endpoint,
    etc..
```

we couldn't use this feature back when we supported python 2 😡

and hey look!
it's [recommended in our developer guide](https://github.com/awslabs/aws-crt-python/blob/main/guides/dev/writing-code.md#functions-with-lots-of-arguments)!